geotrek/api/v2/views/signage.py

```diff
@@ -26,6 +26,11 @@ class SignageTypeViewSet(api_viewsets.GeotrekViewSet):
     queryset = signage_models.SignageType.objects.all().order_by('pk')
 
 
+class SignageConditionViewSet(api_viewsets.GeotrekViewSet):
+    serializer_class = api_serializers.SignageConditionSerializer
+    queryset = signage_models.SignageCondition.objects.all().order_by('pk')
```

Comment:
just keep order by label defined in model.Meta

Reply:
Corrigé